src/commands/utils/helper/kubeloginDownload.ts

```diff
@@ -0,0 +1,137 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as download from '../download/download';
+import * as path from 'path';
+import { getKubeloginConfig } from '../config';
+import { Errorable, failed } from '../errorable';
+import { moveFile } from 'move-file';
+import { longRunning } from '../host';
+
+function getBaseInstallFolder(): string {
+   return path.join(os.homedir(), `.vs-kubernetes/tools`);
```

Comment:
My bad. Should be
`return path.join(os.homedir(), `.vs-kubernetes/tools/kubelogin`);`